packages/ember-routing/lib/system/route.js

```diff
@@ -1616,7 +1616,9 @@ var Route = EmberObject.extend(ActionHandler, Evented, {
     needs access to a parent route's model in order to
     resolve a model (or just reuse the model from a parent),
     it can call `this.modelFor(theNameOfParentRoute)` to
-    retrieve it.
+    retrieve it. Returns an object, not a promise, 
```

Comment:
The general idea is that the thing returned here is the same thing you would refer to in this routes controller or template as `model`. We generally keep the API docs positive in nature (meaning we prefer to say what it does return vs what it does not return).

How about:

> Returns the result of the model hooks promise.

Or:

> Returns the resolved result of a promise from the model hook